### PR TITLE
UPSTREAM: <carry>: Dockerfile: use centos:stream9

### PR DESCRIPTION
**Pull request by kikisdeliveryservice**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: openshift-hack/images/os/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# fedora:29 is an image built within the release scripts:
-# https://github.com/openshift/release/blob/b45a09d248b8cdb8fe3bf5f3cfa0b4fee57d04c8/ci-operator/config/openshift/kubernetes/openshift-kubernetes-release-4.10.yaml#L63-L65
-FROM fedora:29 AS build
+FROM centos:stream9 AS build
```
Comment (cgwalters) [inline openshift-hack/images/os/Dockerfile:1]: I think this needs to be `FROM quay.io/centos/centos:stream9 as build`.  The images on the Docker hub seem not to be updated for CentOS Stream.

Comment (kikisdeliveryservice) [reply openshift-hack/images/os/Dockerfile:1]: Thanks! I'll give it a try!

**Comment by kikisdeliveryservice** [comment]
arrgh:
`Error: Unknown repo: 'built'`

**Comment by kikisdeliveryservice** [comment]
Really weird that this is also hitting some of same errors as https://github.com/openshift/kubernetes/pull/1360
Could changing builder from F29 cause these failures? :thinking:

**Comment by kikisdeliveryservice** [comment]
some changes in ocp/release trying again
/retest-required

**Review by soltysh — APPROVED** [review]
/lgtm
/approve
/remove-label backports/unvalidated-commits
/label backports/validated-commits

**Comment by soltysh** [comment]
To prevent the bot from chasing with us I've opened https://github.com/openshift/release/pull/32444 and I'm overriding the failed jobs which failed due to missing `base-machine-with-rpms` added in that PR
/override ci/prow/e2e-aws-crun
/override ci/prow/e2e-aws-serial 
/override ci/prow/e2e-gcp-upgrade

**Comment by soltysh** [comment]
/override ci/prow/e2e-aws-csi 
/override ci/prow/4.11-upgrade-from-stable-4.10-e2e-aws-ovn-upgrade 
/override ci/prow/e2e-agnostic-cmd

**Pull request merged**